ibis/tests/sql/conftest.py

```diff
@@ -4,6 +4,60 @@
 from ibis.backends.base.sql.compiler import Compiler, QueryContext
 from ibis.tests.expr.mocks import MockBackend
 
+# from ibis.expr.decompile import decompile
+
+# def sql_golden_check(funcname, sql):
+#     if not isinstance(sql, str):
+#         sql = Compiler.to_sql(sql) + '\n'
+
+#     p = Path(ibis.__file__).parent / "tests" / "sqlgolden" / f"{funcname}.sql"
+#     if p.exists():
+#         assert sql == p.read_text()
+#     else:
+#         p.write_text(sql)
+
+
+# def ibis_golden_check(funcname, expr):
+#     if isinstance(expr, str):
+#         return
+
+#     try:
+#         src = decompile(expr, format=True)
+#     except Exception as e:
+#         print(f"Failed to decompile {expr} with error {e}")
+#         return
+
+#     p = Path(ibis.__file__).parent / "tests" / "ibisgolden" / f"ibis_{funcname}.py"
+#     p.write_text(src)
+
+#     try:
+#         locals_ = {}
+#         exec(src, {}, locals_)
+#         restored = locals_['result']  # noqa: F841
+#     except Exception as e:
+#         print(f"Failed to execute decompiled {src} with error {e}")
+#         return
+
+#     # assert expr.equals(restored)
+
+
+# def sqlgolden(func):  # decorator
+#     @functools.wraps(func)
+#     def _decorated(*args, **kwargs):
+#         sql_golden_check(func.__name__, func(*args, **kwargs))
+#         ibis_golden_check(func.__name__, func(*args, **kwargs))
+
+#     return _decorated
+
+
+# def sqlgoldens(func):  # decorator
```

Comment:
I think all this commented out code can be deleted.

Reply:
Removed.